lib/modules/platform/gitea/index.spec.ts

```diff
@@ -192,6 +193,7 @@ describe('modules/platform/gitea/index', () => {
   beforeEach(async () => {
     jest.resetModules();
```

Comment:
this was causing your issue, so you need to use dynamic import to get the right memcache 🙃